File: Dataset/Leetcode/train/75/492.java
 public void XXX(int[] nums) {
        int redIdx = 0;
        int blueIdx = nums.length - 1;

        for (int k = 0; k < nums.length; k++) {
            switch (nums[k]) {
                case 0:
                    redIdx++;
                    break;
                case 2:
                    blueIdx--;
                default:
                    break;
            }
        }

        for (int k = 0; k < redIdx; k++) {
            nums[k] = 0;
        }

        for (int k = redIdx; k <= blueIdx; k++) {
            nums[k] = 1;
        }
        for (int k = blueIdx + 1; k < nums.length; k++) {
            nums[k] = 2;
        }

    }

File: Dataset/Leetcode/test/26/492.cpp
 class Solution {
public:
    int XXX(vector<int>& nums) {
        map<int,int> sortMap;
        for(int i = 0; i < nums.size(); i++){
            sortMap[nums[i]]++;
        }
        map<int,int>::iterator iter = sortMap.begin();
        for(int i = 0; i < sortMap.size(); i++){
            nums[i] = iter->first;
            iter++;
        }
        int count = sortMap.size();
        return count;
    }
};